### PR TITLE
Search function

### PR DESCRIPTION
**Pull request by Will-78**
Fixes to prior bugs and test script.

### CODE REVIEW

File: web/src/components/Search.js [hunk 1]
```diff
@@ -14,22 +14,21 @@ const Search = () => {
...
+            await handleSearch(newValue);
         }
     }
 
     // Define a function to call setSearch function to update the search variable
     const handleSearch = async (input) => {
 
-        // prevents enter from refreshing the page
-        input.preventDefault(); 
+        console.log(input);
```
Comment (jeffreyHoelzel) [inline web/src/components/Search.js]: Not needed anymore since search is working fine

**Review by jeffreyHoelzel — CHANGES_REQUESTED** [review]
Just remove the console.log call since the search function works fine. All tests passed and search function works great. Well done Will!

File: web/src/components/Search.js (hunk 1 above)
Comment (Will-78) [reply web/src/components/Search.js]: Good catch i def forgot.

**Review by jeffreyHoelzel — APPROVED** [review]
Great work Will!

**Pull request merged**